app/(dashboard)/dashboard/_components/ProtocolsTable/ProtocolsTable.tsx

```diff
@@ -1,19 +1,71 @@
+'use client';
+
 import { DataTable } from '~/components/DataTable/DataTable';
 import { ProtocolColumns } from './Columns';
-import { trpc } from '~/app/_trpc/server';
-
-export const ProtocolsTable = async () => {
-  const protocols = await trpc.protocol.get.all.query(undefined, {
-    context: {
-      revalidate: 0,
+import { trpc } from '~/app/_trpc/client';
+import { DeleteProtocol } from '~/app/(dashboard)/dashboard/_components/ProtocolsTable/DeleteProtocols';
+import { useState } from 'react';
+import type { ProtocolWithInterviews } from '~/shared/types';
+import ImportProtocolModal from '../../protocols/_components/ImportProtocolModal';
```

Comment:
Everything looks good, but it would be nice to use import aliases to make code well readable

Reply:
resolved with [fda60e8](https://github.com/complexdatacollective/Fresco/pull/26/commits/fda60e84e200d33d5965f2b833a916fe2d12846f)